src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsResponseDetails.java

```diff
@@ -35,49 +35,18 @@ public static List<FeedbackResponseAttributes.UpdateOptions> getUpdateOptionsFor
 
         FeedbackResponseDetails details;
         FeedbackRankRecipientsResponseDetails responseDetails;
-        boolean[] isRankUsed;
+        // boolean[] isRankUsed;
```

Comment:
Can this value commented out statement be deleted?

Reply:
Sorry for the late reply. Yes, it can be removed.

Reply:
@gustafssonlinnea As you have already mentioned, this can be removed, so please remove it. We would prefer not to leave commented out code in the codebase.

Other than that, code looks alright.

Reply:
Hi @gustafssonlinnea , As mentioned, could you please remove this commented line of code?